draft-ietf-elegy-rfc8989bis.md

```diff
@@ -178,12 +178,10 @@ attacker must produce 365 volunteers to exceed a 50% chance of NomCom capture
 (see {{capture-math}}).
 
 A sudden surge in the number of volunteers, particularly of people that no one
-recognizes as a part of the community, is an early-warning sign for the
-community, leadership and the IETF Secretariat to further investigate, and where
-needed take action as defined in  RFC 8713 Section 3.7.3 to invalidate such
-candidates. The community should monitor and assess a sudden increase in the
-number of online registration fee waivers awarded in accordance with
-{{Section 4 of I-D.ietf-shmoo-remote-fee}}.
+recognizes as a part of the community, is an early-warning sign. Anyone with
+concerns about the integrity of the process should bring those concerns to the
+IESG to further investigate,and where needed take action as defined in RFC 8713
```

Comment:
This might be more clear: "IESG for further investigation and to take action, if needed, as defined in RFC 8713"  If not, fine, but at least a space after the comma :)